### ISSUE TITLE
genquery ignores `--consistent_labels`

### ISSUE THREAD

**Issue by tpasternak** [bazelbuild/bazel#24325]
### Description of the bug:

```py
# A simple genrule that generates a file using echo
genrule(
    name = "generate_hello_file",
    srcs = [],
    outs = ["hello.txt"],
    cmd = "echo 'Hello, Bazel!' > $@",
)

# A genquery rule to query all files in the current package
genquery(
    name = "query_hello_files",
    scope = ["@@//:generate_hello_file"],
    expression = "//:generate_hello_file",
    opts = ["--output=streamed_jsonproto", "--consistent_labels=true"],
)
```

In the output of `bazel build :query_hello_files`, the labels not prefixed with `@@`

### Which operating system are you running Bazel on?

Fedora 40

### What is the output of `bazel info release`?

release 7.4.1

**Comment by fmeum** [comment]
@tpasternak Could you test https://github.com/bazelbuild/bazel/pull/24326?

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 8.0.0 RC3](https://github.com/bazelbuild/bazel/releases/tag/8.0.0rc3). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting USE_BAZEL_VERSION=8.0.0rc3. Thanks!
